lib/cluster.js

```diff
@@ -341,9 +341,9 @@ function masterInit() {
       process: workerProcess
     });
 
-    worker.on('message', (message, handle) =>
-      cluster.emit('message', message, handle)
-    );
+    worker.on('message', function(message, handle) {
+      cluster.emit('message', this, message, handle);
```

Comment:
The event is emitted on the worker object, the listener simply re-emits it on the cluster object (hence why it was changed from an arrow function to a normal function - no lexical this.)